### ISSUE TITLE
Confused about quering many to many relation over api

### ISSUE THREAD

**Issue by ampedweb** [directus/directus#2420]
Hi guys, wouldn't say this is a bug, more likely a lack of understanding on my part!

I'm struggling to understand how I retrieve Many to Many relations over the API. 
My scenario is pretty simple so far: I have table `page` a junction table `page_prize` and a table `prize`

I've managed to get the relationship setup correctly (I think!) via the Directus App and related a prize to this page (see screenshot):

![image](https://user-images.githubusercontent.com/22029493/63379834-0ef12300-c38d-11e9-9f77-7283310457e4.png)

My question is, how do I for example query a specific page like this:

http://mytestapi.local/_/items/page/1

And also retrieve all of the `prize` records relating to this particular page?

I've read through the docs and I can't seem to see a M2M example? Unless I'm being blind, in which case I apologise! :)

**Comment by rijkvanzanten** [comment]
Heya! 

By default, the Directus API will only return data one "level" deep in order to preserve performance and bandwidth usage (fetching _every_ relationship at all times can result in circular dependencies and gigantic JSON structures). There is a parameter you can use to choose what fields to return. This parameter can also be used to choose what relational data to return. https://docs.directus.io/api/reference.html#fields

TL;DR

Add the `fields` parameter: `fields=*,related_prizes.*.*`

**Comment by ampedweb** [comment]
Hi there! Thanks for your reply. Unfortunately I've already tried that and it still doesn't seem to work. The url I'm using is:
`http://mytestapi.local/_/items/page/1?fields=*,related_prizes.*.*`

And I just get back the first level page, despite there definitely being a related prize attached to the homepage?

`{
    "data": {
        "id": 1,
        "status": "published",
        "created_by": {
            "id": 1
        },
        "created_on": "2019-08-18 15:52:51",
        "modified_by": {
            "id": 1
        },
        "modified_on": "2019-08-21 06:43:30",
        "meta_title": "Test meta Title",
        "meta_description": "Hello",
        "title": "Homepage"
    }
}`

**Comment by rijkvanzanten** [comment]
Can you make sure that the role you're using to access this endpoint (seems to be the `public` role) has permission to read these items? If it does, can you try `fields=*.*.*` instead?

**Comment by ampedweb** [comment]
That's exactly what it was! I knew it was me just missing something obvious. Thanks very much for your help :)
